warp/Network/Wai/Handler/Warp/Types.hs

```diff
@@ -1,16 +1,13 @@
 {-# LANGUAGE DeriveDataTypeable #-}
 {-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE CPP #-}
 
 module Network.Wai.Handler.Warp.Types where
 
 import qualified UnliftIO
 import qualified Data.ByteString as S
 import Data.IORef (IORef, readIORef, writeIORef, newIORef)
 import Data.Typeable (Typeable)
-#ifdef MIN_VERSION_x509
 import Data.X509
-#endif
```

Comment:
@kazu-yamamoto 

Interestingly I get this error when trying to depend on Warp with the x509 flag _disabled_:

```
Network/Wai/Handler/Warp/Types.hs:10:1: error:
    Could not load module ‘Data.X509’
    It is a member of the hidden package ‘x509-1.7.7’.
    Perhaps you need to add ‘x509’ to the build-depends in your .cabal file.
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
10 | import Data.X509
   | ^^^^^^^^^^^^^^^^
```